network/README.md

```diff
@@ -92,71 +91,53 @@ Some peers aren't discovered through the `PeerList` messages exchanged through p
 sequenceDiagram
 Node ->> Peer-1: Handshake - v1.9.5
 Peer-1 ->> Node: PeerList - Peer-2
-Node ->> Peer-1: ACK - Peer-2
 Note left of Node: Node is connected to Peer-1 and now tries to connect to Peer-2.
 Node ->> Peer-2: Handshake - v1.9.5
 Peer-2 ->> Node: PeerList - Peer-1
-Node ->> Peer-2: ACK - Peer-1
 Note left of Node: Peer-3 was never sampled, so we haven't connected yet!
 Node --> Peer-3: No connection
 ```
 
-To guarantee that a node can discover all peers, each node periodically gossips a sample of the peers it knows about to other peers.
+To guarantee that a node can discover all peers, each node periodically sends a `GetPeerList` message to a random peer.
 
 ##### PeerList Gossip
 
 ###### Messages
 
-A `PeerList` is the message that is used to communicate the presence of peers in the network. Each `PeerList` message contains networking-level metadata about the peer that provides the necessary information to connect to it, alongside the corresponding transaction id that added that peer to the validator set. Transaction ids are unique hashes that only add a single validator, so it is guaranteed that there is a 1:1 mapping between a validator and its associated transaction id.
+A `GetPeerList` message requests that the peer sends a `PeerList` message. `GetPeerList` messages contain a bloom filter of already known peers to reduces useless bandwidth on `PeerList` messages.
```

Comment:
nit: `reduce`

Reply:
nit: may want to be a little more explicit as to why this "reduces useless bandwidth" (i.e. only send peers that aren't in the bloom filter)